examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # Update equations (3rd-order ADER timestepping)
</pre><p>I would prefer writing the PDE and use solve rather than by hand. It would also make the method more explicit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>